src/ssl/helper.cc

```diff
@@ -166,7 +166,7 @@ Ssl::HandleGeneratorReply(void *data, const ::Helper::Reply &reply)
 }
 #endif //USE_SSL_CRTD
 
-helper::Pointer Ssl::CertValidationHelper::ssl_crt_validator = nullptr;
+Helper::Client::Pointer Ssl::CertValidationHelper::ssl_crt_validator = nullptr;
```

Comment:
`Pointer` do not need explicit nullptr assignment.
```suggestion
Helper::Client::Pointer Ssl::CertValidationHelper::ssl_crt_validator;
```

Reply:
Done.